rosbag2_storage/test/test_plugin.hpp

```diff
@@ -32,13 +33,15 @@ class TestPlugin : public rosbag2_storage::storage_interfaces::ReadWriteInterfac
 
   rosbag2_storage::BagInfo info() override;
 
-  bool has_next() const override;
+  void create_topic(const std::string & name, const std::string & type_id) override;
```

Comment:
`topic_id` here is the topic type? Or which id is meant here? If so, I would recommend having it renamed to `type_type` in order to keep the vocabulary in sync with upstream

Reply:
Changed to `topic_name` and `type`

Reply:
I haven't seen `type_type` anywhere but changed the parameters to `name` and `type`.